lib/inbox.rb

```diff
@@ -237,32 +237,45 @@ def latest_cursor
       "message" => Inbox::ExpandedMessage,
     }
 
-    def _build_exclude_types(exclude_types)
-      exclude_string = "&exclude_types="
+    def _build_exclude_types(types)
+      exclude_string = "&exclude_types=" + _build_types_list(types)
+    end
+
+    def _build_include_types(types)
+      include_string = "&include_types=" + _build_types_list(types)
+    end
 
-      exclude_types.each do |value|
+    def _build_types_list(types)
+      type_string = ""
+      types.each do |value|
         count = 0
         if OBJECTS_TABLE.has_value?(value)
           param_name = OBJECTS_TABLE.key(value)
-          exclude_string += "#{param_name},"
+          type_string += "#{param_name},"
         end
       end
 
-      exclude_string = exclude_string[0..-2]
+      type_string = type_string[0..-2]
     end
 
-    def deltas(cursor, exclude_types=[], expanded_view=false)
-      return enum_for(:deltas, cursor, exclude_types, expanded_view) unless block_given?
+    def deltas(cursor, exclude_types=[], include_types=[], expanded_view=false)
```

Comment:
Hmm, If I recall correctly, Ruby does weird thing with positional arguments (i.e: https://eval.in/506574). It would be pretty useful if you could either:
- figure out how to not break backward compatibility
- otherwise refactor the function by using a parameter hash.